src/extension.ts

```diff
@@ -17,9 +17,23 @@ import { MessageHandler, SidebarView } from "@views";
 import { applyManifest, buildOnRightClick, buildProject, runProject, setToken } from "@vscommands";
 import { openAddConnectionsPage } from "@vscommands/sideBarActions";
 import { openBaseURLInputDialog, openWalkthrough } from "@vscommands/walkthrough";
-import { commands, ExtensionContext } from "vscode";
+import { commands, ExtensionContext, workspace, window } from "vscode";
 
 export async function activate(context: ExtensionContext) {
+	workspace.onDidChangeConfiguration(async (event) => {
+		if (event.affectsConfiguration("autokitteh.baseURL")) {
```

Comment:
do you have a way to know if the URL did change? 
if the user just save without a change will it emit an event?

Reply:
Without a change it will not emit an event.